package.json

```diff
@@ -55,7 +55,8 @@
     "semver": "~6.3.0",
     "sharp": "~0.23.1",
     "simpl-schema": "~1.5.6",
-    "stripe": "~5.10.0"
+    "stripe": "~5.10.0",
+    "transliteration": "^2.1.7"
```

Comment:
Remove transliteration, any function that needs it should exist in the `api-utils` package: https://github.com/reactioncommerce/api-utils

Reply:
Done